contracts/OlasMech.sol

```diff
@@ -51,6 +53,7 @@ abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
     // Reentrancy lock
     uint256 internal _locked = 1;
 
+    // TODO do we need the next two maps? Is it for staking activity checks?
```

Comment:
@kupermind we should check if we can remove either this map or the one on the marketplace. Feels like info is being duplicated

Reply:
I believe you are correct here, seems like just the legacy from when mechs were created not with the MM interactions.